contrib/lisp/org-roam-helm-bibtex.el

```diff
@@ -0,0 +1,39 @@
+;;; org-roam-helm-bibtex.el --- Org-roam helm-bibtex integration  -*- fill-column: 78; lexical-binding: t; -*-
+
+;; Authors: Leo Vivier <leo.vivier+dev@gmail.com>
+
```

Comment:
The file needs proper package headers. Declare a dependency on org-roam.

Reply:
I'll just base my header off `org-roam`'s, then.